android/app/build.gradle

```diff
@@ -216,6 +217,11 @@ dependencies {
     implementation('com.crashlytics.sdk.android:crashlytics:2.9.9@aar') {
         transitive = true
     }
+    implementation(project(':react-native-jitsi-meet')) {
```

Comment:
Doesn't it work with auto linking?

Reply:
No, it generates an error because has duplicated modules like `react-native-webview`. :(